flink-core/src/main/java/org/apache/flink/core/fs/FileSystem.java

```diff
@@ -758,11 +758,6 @@ public FSDataOutputStream create(Path f, boolean overwrite) throws IOException {
 	 */
 	public abstract boolean isDistributedFS();
 
-	/**
-	 * Gets a description of the characteristics of this file system.
-	 */
-	public abstract FileSystemKind getKind();
```

Comment:
The `FileSystem` class is a `@Public` API, meaning that no methods may be removed.